tlsobs-api/insert_test.go

```diff
@@ -0,0 +1,134 @@
+package main
```

Comment:
Would this be best placed in a `benchmark` subdirectory of the database folder, since we shouldn't run it as part of the normal `go test` process?